c2corg_ui/static/js/auth/auth.js

```diff
@@ -151,10 +162,35 @@ app.AuthController.prototype.successRegister_ = function(response) {
  * @private
  */
 app.AuthController.prototype.errorRegister_ = function(response) {
-  // TODO
-  alert('register error');
-  console.log('register error');
-  console.log(response);
+  this.alerts_.add({
+    'type': 'danger',
+    'msg': this.formatErrorMsg_(response),
+    'timeout': 5000
+  });
+};
+
+
+/**
+ * @param {Object} response Response from the API server.
+ * @return {string}
+ * @private
+ */
+app.AuthController.prototype.formatErrorMsg_ = function(response) {
+  // TODO: i18n
+  // TODO: filter strings from response using $sanitize or another tool
+  var errors = response['data']['errors'],
+      len = errors.length,
+      msg = '';
+  if (len == 1) {
+    msg = errors[0]['description'];
```

Comment:
As suggested by @gberaudo it would make sense to escape this ``description`` field since the standard escaping AngularJS normally performs on expressions is disabled in the alert partial using the ``appTrustAsHtml`` filter.

We could for instance use ``$sanitize`` but it requires installing a separated package.
https://docs.angularjs.org/api/ngSanitize/service/$sanitize